packages/atomic/src/components/commerce/atomic-commerce-interface/store.ts

```diff
@@ -78,5 +80,9 @@ export function createAtomicCommerceStore(): AtomicCommerceStore {
         makeDesktopQuery(commonStore.state.mobileBreakpoint)
       ).matches;
     },
+
+    getUniqueIDFromEngine(engine: CommerceEngine): string {
+      return responseIdSelectorFromEngine(engine);
```

Comment:
The responseId can come from either the search, product listing or recommendations state. Why are we only using the product listing's state here?

Reply:
But... this is the atomic-store for `atomic-commerce-interface`, which handle both search and listing. 

I don't understand why we must not differentiate between the two types when we need to get a `uniqueIDFromEngine` 🤔